pwa/src/components/common/codeBlock/codeBlock.tsx

```diff
@@ -1,19 +1,27 @@
 import * as React from "react";
 import "./codeBlock.css";
+import Prism from "prismjs";
+import "../../../styles/prism.css";
+import 'prismjs/components/prism-json';
+import 'prismjs/components/prism-yaml';
 
 interface CodeBlockProps {
   code: any,
-  language: "json" | "xml"
+  language: "json" | "xml" | "css" | "html" | "yaml"
```

Comment:
I don't think we're actually ever rendering `css` `html` or `yaml` code. Therefore, I would not add them to the language options here.